README.md

```diff
@@ -1,6 +1,29 @@
-# Portfolio
+# Rhet Pulley - SQL Developer, Data Analyst, Report Writer, ETL Developer
+<img src="https://user-images.githubusercontent.com/108885804/210109116-8a16571a-eb1b-4ac6-941b-74c8c413797c.jpg" height="100px"/>  Rwpull@gmail.com  
+
+
+For the past year I have been able to learn Microsoft SQL, PostgreSQL and MySQL through training with a Senior SQL Developer. 
```

Comment:
Hi, I'm Rhet and I'm an apprentice-level SQL Developer, Data Analyst, Report Writer, ETL Developer etc.  I'm currently looking for my first full-time professional position in the world of Data.  For the past year I've been training myself and working with a Senior SQL developer to learn everything I can about T-SQL (Microsoft SQL), MySQL, PostgreSQL, as well as the fields of data analytics and report writing.  I have experience designing new databases, creating new stored procedures, adding foreign keys and maximizing database performance as well as securing private data.

Maybe something like this?